src/basic/iovec-util.c

```diff
@@ -3,6 +3,16 @@
 #include "iovec-util.h"
 #include "string-util.h"
 
+size_t iovec_total_size(const struct iovec *i, size_t n) {
```

Comment:
While at it, please rename `i` to `iovec`. (`i` really should be used only as an iterator.)